### ISSUE TITLE
Suggestion: Don't randomize the title color 

### ISSUE THREAD

**Issue by aboyher** [atomcorp/themes#4]
I like the app, but i don't like that as i scroll through the color schemes, the color of the title and highlighting surrounding the actual theme colors seems to be randomized. It changed how i perceive the theme. I think I'm getting a theme with a nice red color, but really i'm not.

**Comment by atomcorp** [comment]
Hey! They're not completely random, if that's what you mean, it only picks colours from the selected theme.

**Comment by dominik-ba** [comment]
I agree. Although you only pick colors from the selected theme I was very confused by the different title colors just by refreshing/switching the page.
Especially since it is not explained on the page neither you easily see that the colors are only picked from the theme. Still with having read this I still don't know what colors especially are used for the font. You use a random foreground color or any color or something else?

For all those reasons I think it would be easier without randomization.

**Comment by atomcorp** [comment]
Hey thanks for the feedback!

I do think that ultimately the switching colours in the UI thing is too distracting from the actual purpose of the app, and I have warmed up to the idea of making it look more grown-up. I made it as a bit of fun for myself, but it seems like people are starting to use it (which is ace), so it should look a bit more utilitarian. I can always include the colour switching in a more subtle way.

I have UI refresh almost done, which has some good improvements, but also keeps the same colour switching concept. I might release it before tackling this issue, but I will definitely do this at some point soon.

Incidentally, the reason it's randomised is there aren't really any consistent colour combinations with good contrast, aside from maybe background/foreground. So I have to filter every colour for reasonable contrast against it's background. 

Thanks again!
